function-server/src/main/java/io/dispatchframework/javabaseimage/SimpleFunctionExecutor.java

```diff
@@ -67,14 +75,40 @@ public String execute(String message) {
             }
 
             if (err == null) {
+                Map<Object, Object> context = (Map<Object, Object>) req.getContext();
+                Object payload = req.getPayload();
+                Double timeout = (Double) context.getOrDefault("timeout", 0);
```

Comment:
I added the cast to Double because it failed with Integer against a real dispatch function execution. This might be because I used int64 for the timeout value in the function-manager.